HW_1/Int_2.py

```diff
@@ -0,0 +1,20 @@
+"""
+Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+*Пример:*
+
+6 -> 1  4  1
+24 -> 4  16  4
+    60 -> 10  40  10
+"""
+q = int(input('Введите число: '))
+
+w = int((q/3)*2)
+
+e = int((w/2)/2)
+
+r = int(e)
```

Comment:
Во первых, не помешало бы проверить входные данные на кратность 6. Во-вторых, у операции деления и умножения одинаковый приоритет. В-третьих, функцию `int` использовать необязательно, можно использовать целочисленное отделение. Ну и конечно, не будем забывать про пробелы.
```suggestion
w = q // 3 * 2

e = r = w // 2 // 2
```